modules/applications/pages/faq.adoc

```diff
@@ -1,7 +1,23 @@
 = FAQ
-:description:
+:description:Find here the answers to the frenquently asked questions about Bonita UI Builder and Application Links.
+
+== My application link is incorrect when accessing the Application Directory from my custom application.
+
+image::images/app-link/incorrectAppLink.gif[Application has incorrect Link]
+
+If you have customized e.g. the Bonita User Application, you may be using an old version of the layout.
+
+image::images/app-link/issueOldLayout.png[Application with old layout]
+
+In that case, the Application Directory opened from that application only will not display Application Links correctly, but the same way as Legacy Applications. +
+Fix this by using `custompage_layoutBonita` as the layout value, which will automatically migrate to the latest layout.
+
+You may also encounter this issue if you use a xref:applications:customize-layouts.adoc[customized layout] instead of the official one. In that case, update your custom layout by re-applying your modifications to the latest layout / application directory version that implements the improvements for Application Links.
```

Comment:
```suggestion
You may also encounter this issue if you use a xref:applications:customize-layouts.adoc[customized layout] instead of the official one. In that case, update your application's layout by re-applying your modifications to the latest layout version that implements the improvements for Application Links.
```